src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -966,6 +1011,13 @@ public ShopifyCustomer getCustomer(final String customerId) {
 		return getCustomerSavedSearches(response);
 	}
 
+	public int getCustomerSavedSearchesCount(final String pageInfo) {
```

Comment:
Why does this take pageInfo as parameter?
This does not require any parameters according to the [API reference](https://shopify.dev/api/admin-rest/2022-01/resources/customersavedsearch#[get]/admin/api/2022-01/customer_saved_searches/count.json)